src/main/java/com/comphenix/protocol/events/PacketContainer.java

```diff
@@ -17,17 +17,26 @@
 
 package com.comphenix.protocol.events;
 
+import com.comphenix.protocol.reflect.accessors.Accessors;
```

Comment:
nit: i would prefer the import order be separated by tld with java first then in alphabetical order

Reply:
should be done